yaml-tests/src/test/resources/transactions-tests.yamsql

```diff
@@ -37,21 +37,29 @@
 #       whatever statements they want in whichever location.
 ---
 options:
-    supported_version: 4.3.5.0
+    supported_version: !current_version
 ---
 schema_template:
-    create table t1(id bigint, col1 bigint, primary key(id))
+    create table table_t1(id bigint, col1 bigint, primary key(id))
+---
+transaction_setups:
+  t1_less_than_50: create temporary function t1() on commit drop function AS SELECT * FROM table_t1 where id < 50
```

Comment:
In theory, it wouldn't be hard to change this to allow an array of strings if there's a bunch of setup steps